rocky/rocky/views/ooi_view.py

```diff
@@ -37,6 +37,7 @@ def setup(self, request, *args, **kwargs):
         self.filtered_ooi_types = request.GET.getlist("ooi_type", [])
         self.clearance_levels = request.GET.getlist("clearance_level", [])
         self.clearance_types = request.GET.getlist("clearance_type", [])
+        self.search_string = request.GET.get("search", "")
 
     def get_active_filters(self) -> dict[str, str]:
```

Comment:
I have realised theta we need to add to the active filters, that the user is searching for the active filters. I will add this myself.